src/utilities/fetch-package-repos.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const path = require('path');
+const _ = require('lodash');
+const GithubAPI = require('@octokit/rest');
+
+const fetch = {
+  loaders: [
+    {
+      organization: 'webpack-contrib',
+      suffixes: ['-loader']
+    },
+    'babel/babel-loader',
+    'postcss/postcss-loader',
+    'peerigon/extract-loader'
+  ],
+
+  plugins: [
+    {
+      organization: 'webpack-contrib',
+      suffixes: ['-webpack-plugin', '-extract-plugin']
+    }
+  ]
+};
+
+const api = new GithubAPI();
+
+async function paginate (org) {
+  let response = await api.repos.getForOrg({ org, type: 'public', per_page: 100});
+  let {data} = response;
+
+  while (api.hasNextPage(response)) {
+    response = await api.getNextPage(response);
+    data = data.concat(response.data);
+  }
+
+  return data;
+}
+
+async function main() {
+  if (process.env.GITHUB_TOKEN) {
```

Comment:
I am not sure about leaving this here, this solution doesn't work because Travis doesn't export ENV variables to PRs from other repos (security measure), and deploying master wouldn't consume API limit.

Reply:
I'll remove that bit

Reply:
This specific script is not supposed to be run on travis anyway, so it doesn't really matter